keras_nlp/layers/transformer_decoder.py

```diff
@@ -292,19 +307,23 @@ def call(
             cross_attended = self._cross_attention_dropout(
                 cross_attended,
             )
-            attention_output = self._add_and_norm(
-                cross_attended,
-                attention_output,
-                self._cross_attention_layernorm,
-            )
-
+            attention_output = residual_attention_output + cross_attended
+            if not self.normalize_first:
+                attention_output = self._cross_attention_layernorm(
+                    attention_output
+                )
+
+        residual_attention_output = attention_output
+        if self.normalize_first:
+            attention_output = self._feedforward_layernorm(attention_output)
```

Comment:
feedforward -> feed_forward in this name to match others

Reply:
"Feedforward" is one word. So, I'll change `feed_forward` to `feedforward`.

Reply:
Sg! As long as we are consistent (preferably across decoder and encoder).